docs/ops/filesystems.md

```diff
@@ -112,10 +111,9 @@ To prevent inactive streams from taking up the complete pool (preventing new con
 `fs.<scheme>.limit.stream-timeout`. If a stream does not read/write any bytes for at least that amount of time, it is forcibly closed.
 
 These limits are enforced per TaskManager, so each TaskManager in a Flink application or cluster will open up to that number of connections.
-In addition, the The limit are also enforced only per FileSystem instance. Because File Systems are created per scheme and authority, different
+In addition, the limit are also enforced only per FileSystem instance. Because File Systems are created per scheme and authority, different
```

Comment:
either `the limit is` or the `the limits are`

also, `only` should be placed before `enforced`.